### ISSUE TITLE
Processing images

### ISSUE THREAD

**Issue by Jalkhov** [dothinking/pdf2docx#57]
I have clear limitations of `pdf2docx`, but with I understand very well the part of 

> No floating images

How exactly are the **floating images**? My doubt arises because I have processed two files with images, one with more images than the other but both have, however with one I throw an error, and with the other not, I attach the files and error information.

**File that throws error: ValueError: no valid image found**
[test.pdf](https://github.com/dothinking/pdf2docx/files/5415462/test.pdf)

**File processed correctly**
[test2.pdf](https://github.com/dothinking/pdf2docx/files/5415466/test2.pdf)

<hr>

I assume the error occurs on page 1 because the output console shows me:

`Processing Pages: 1/16...Traceback (most recent call last):`

So apparently that's as far as processing goes.

<hr>

Beyond simply making an `exception` when there are errors with the images, something like that:

```
from pdf2docx import parse

try:
	parse('Input.pdf', 'Output.docx')
except ValueError:
	print('Image error, this file cant be processed.')
```

I would like to know exactly how to identify a **floating image** in a PDF.

**Comment by dothinking** [comment]
> How exactly are the floating images? 

It's a floating image If it overlaps with other elements like text or other images. Since they can't be arranged in flow mode -> one of them is floating. For instance, a background image behind some texts. 

> ValueError: no valid image found

So this has nothing to do with floating image since no overlaps exist in Page 1. It seems an upstream issue -> `pdf2docx` relies on `PyMuPDF`, a powerful PDF processing library, to extract texts and images. I raised an issue to `PyMuPDF`. Hope you don't mind since I used the image in test.pdf for sample.

https://github.com/pymupdf/PyMuPDF/issues/699
